src/Core/Components/List/FluentCombobox.razor.cs

```diff
@@ -138,16 +138,20 @@ protected override async Task ChangeHandlerAsync(ChangeEventArgs e)
             if (item is null)
             {
                 SelectedOption = default;
-                await base.ChangeHandlerAsync(e);
             }
             else
             {
                 await OnSelectedItemChangedHandlerAsync(item);
             }
-
         }
     }
 
+    protected override async Task InputHandlerAsync(ChangeEventArgs e)
+    {
+        await base.ChangeHandlerAsync(e);
```

Comment:
Why did you remove the usage of `Immediate`?
The method `FluentInputBase.InputHandlerAsync` will not used.

Reply:
Fixed in last commit